cmd/umserver/main.go

```diff
@@ -70,8 +72,22 @@ func main() {
 	}()
 	logger.LogUM.Info("Server Listening at %s...", srv.Addr)
 
-	// TODO: There will be actual information about PostgreSQL connection in future
-	// ...
+	dbConfig, err := cfg.DBConfig(ctx)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
It would be great to add context about error

Reply:
Done